haystack/modeling/model/optimization.py

```diff
@@ -161,13 +133,13 @@ def initialize_optimizer(
         schedule_opts["num_training_steps"] = num_train_optimization_steps
 
     # Log params
-    tracker.track_params({"use_amp": use_amp, "num_train_optimization_steps": schedule_opts["num_training_steps"]})
```

Comment:
I moved tracking of this parameter into the Trainer since `use_amp` is no longer needed or used by the optimizer. Is that alright?